modules/nf-core/bclconvert/tests/main.nf.test

```diff
@@ -4,11 +4,12 @@ nextflow_process {
     script "../main.nf"
     process "BCLCONVERT"
     config "./nextflow.config"
+
     tag "bclconvert"
     tag "modules"
     tag "modules_nfcore"
 
-    test("homo sapiens illumina [bcl]") {
+    test("homo_sapiens illumina [bcl]") {
         when {
             process {
                 //TODO use new test dataset when available, see https://github.com/nf-core/test-datasets/issues/996
```

Comment:
Looks like this has been implemented:
https://github.com/nf-core/test-datasets/pull/1082
(don't have to do it now, but in case you fancy doing it while modifying the module anyway)